exchanges/binance/binance.go

```diff
@@ -746,6 +746,24 @@ func (b *Binance) WithdrawCrypto(asset, address, addressTag, name, amount string
 	return resp.ID, nil
 }
 
+// WithdrawStatus gets the status of recent withdrawals
+func (b *Binance) WithdrawStatus(c currency.Code) ([]WithdrawStatusResponse, error) {
```

Comment:
While in the context of the rpc function we don't use it yet, I would appreciate if you filled out the functionality of this implementation to match Binance's wAPI https://github.com/binance-exchange/binance-official-api-docs/blob/master/wapi-api.md#withdraw-history-user_data
ie add `startTime`, `endTime`, `status`